spark/src/main/java/org/apache/iceberg/spark/data/vectorized/RowPositionColumnVector.java

```diff
@@ -27,11 +27,11 @@
 import org.apache.spark.sql.vectorized.ColumnarMap;
 import org.apache.spark.unsafe.types.UTF8String;
 
-public class RowPostitionColumnVector extends ColumnVector {
+public class RowPositionColumnVector extends ColumnVector {
 
   private final long batchOffsetInFile;
 
-  RowPostitionColumnVector(long batchOffsetInFile) {
+  RowPositionColumnVector(long batchOffsetInFile) {
```

Comment:
Can we fix typos in a separate PR? This is already touching quite a few files and this file doesn't need to change.